src/styles/block/card.scss

```
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

That should be moved to nomalize.scss (same for all classes)
```suggestion
* {
  box-sizing: border-box;
}
```